src/components/OpmSideBar.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from "react";
+import CustomSideBar from "../components/CustomSideBar";
+import { FETCH_TYPES } from "../components/utils/Utils";
+import OPM from "../views/opm";
+import { LOCALE_OPTIONS, OPM_CHANNELS, PAYMENT_TYPES } from "../constants/appConstants";
+import { OpmSideBarProps } from "../@types/components/commonTypes";
+
+const OpmSideBar: FC<OpmSideBarProps> = (props) => {
```

Comment:
`const OpmSideBar = (props: OpmSideBarProps) => {`
use this pattern instead.
**test the code.**